package.json

```diff
@@ -42,6 +42,7 @@
   "devDependencies": {
     "babel-cli": "^6.7.5",
     "babel-eslint": "^7.1.1",
+    "babel-plugin-transform-object-rest-spread": "6.22.0",
```

Comment:
I'm not sure - I don't see that error locally with node 7.5.0 and npm 4.2.0. Maybe try wiping out `node_modules` and reinstalling? If that doesn't work please post your node/npm versions.